src/soilbiogeochem/SoilBiogeochemNitrogenStateType.F90

```diff
@@ -503,6 +722,87 @@ subroutine Restart ( this,  bounds, ncid, flag, totvegc_col )
+                     dim1name='column', long_name='', units='', &
+                     interpinic_flag='interp', readvar=readvar, data=this%tan_f3_col)
+
+       call restartvar(ncid=ncid, flag=flag, varname='fert_u0', xtype=ncd_double, &
+                     dim1name='column', long_name='', units='', &
+                     interpinic_flag='interp', readvar=readvar, data=this%fert_u0_col)
+       call restartvar(ncid=ncid, flag=flag, varname='fert_u1', xtype=ncd_double, &
+                     dim1name='column', long_name='', units='', &
+                     interpinic_flag='interp', readvar=readvar, data=this%fert_u1_col)
+
+       call restartvar(ncid=ncid, flag=flag, varname='man_u_grz', xtype=ncd_double, &
+                     dim1name='column', long_name='', units='', &
+                     interpinic_flag='interp', readvar=readvar, data=this%man_u_grz_col)
+       call restartvar(ncid=ncid, flag=flag, varname='man_a_grz', xtype=ncd_double, &
+                     dim1name='column', long_name='', units='', &
+                     interpinic_flag='interp', readvar=readvar, data=this%man_a_grz_col)
+       call restartvar(ncid=ncid, flag=flag, varname='man_r_grz', xtype=ncd_double, &
+                     dim1name='column', long_name='', units='', &
+                     interpinic_flag='interp', readvar=readvar, data=this%man_r_grz_col)
+
+       call restartvar(ncid=ncid, flag=flag, varname='man_u_app', xtype=ncd_double, &
+                     dim1name='column', long_name='', units='', &
+                     interpinic_flag='interp', readvar=readvar, data=this%man_u_app_col)
+       call restartvar(ncid=ncid, flag=flag, varname='man_a_app', xtype=ncd_double, &
+                     dim1name='column', long_name='', units='', &
+                     interpinic_flag='interp', readvar=readvar, data=this%man_a_app_col)
+       call restartvar(ncid=ncid, flag=flag, varname='man_r_app', xtype=ncd_double, &
+                     dim1name='column', long_name='', units='', &
+                     interpinic_flag='interp', readvar=readvar, data=this%man_r_app_col)
+
+       call restartvar(ncid=ncid, flag=flag, varname='man_tan_stored', xtype=ncd_double, &
+                     dim1name='column', long_name='', units='', &
+                     interpinic_flag='interp', readvar=readvar, data=this%man_tan_stored_col)
+       call restartvar(ncid=ncid, flag=flag, varname='man_n_stored', xtype=ncd_double, &
+                     dim1name='column', long_name='', units='', &
+                     interpinic_flag='interp', readvar=readvar, data=this%man_n_stored_col)
+       call restartvar(ncid=ncid, flag=flag, varname='fan_grz_fract', xtype=ncd_double, &
+                     dim1name='column', long_name='', units='', &
+                     interpinic_flag='interp', readvar=readvar, data=this%fan_grz_fract_col)
+
+    end if
+    call restartvar(ncid=ncid, flag=flag, varname='fan_totn', xtype=ncd_double, &
```

Comment:
within if statement above?

Reply:
- [ ] @juliusvira can you address this question?

Reply:
We are keeping fan_totn defined but zero when FAN is off, so it should keep outside the if.